### ISSUE TITLE
[FEAT EXTRACTION] Will updating 3D Slicer Radiomics extension from 2.2 to 3.0.1 affect the calculation result?

### ISSUE THREAD

**Issue by ucs198604** [AIM-Harvard/pyradiomics#711]
Hi, 
Will updating 3D Slicer Radiomics extension from 2.2 to 3.0.1 affect the calculation result?
According to your release note, there was a bug pertaining to GLSZM calculation, which resulted in errors (and was fixed at version 3.0.1)
So... if there is no error occurs, can I say that the radiomics results we get from version 2.2 would be the same as what we get from version 3.0.1?

Thank you.

**Comment by JoostJM** [comment]
Not necessarily. Review the changelog in the pyradiomics documentation. Amy changes affecting calculated results are detailed there.
